scripts/cicd.sh

```diff
@@ -35,7 +38,7 @@ publish?() {
     # Figure out whether to release the docker image and executable binary
     #   The lack of setting PUBLISH to anything means its undefined
 
-    tag_of_current_commit="$(git describe --exact-match --tags HEAD || true)"
+    tag_of_current_commit="$(git describe --exact-match --tags HEAD &>/dev/null || true)"
```

Comment:
Should this be `2>/dev/null` or are you looking for a boolean not a tag(?) .. Variable seems oddly named if you want a boolean.

Reply:
Great catch, thank you.